lib/types/interfaces.ts

```diff
@@ -4,13 +4,16 @@ import winston from "winston";
 export interface IafUploadModule {
     logger: winston.Logger;
     playlistName: string;
-    onFileAdd(filePath: string, readStream: Readable): any; 
+    outputBucket: string;
```

Comment:
I don't want to clutter the generic interface with AWS specifics. Can these things be provided in the constructor to the AWS plugin?
Consider writing an IAF plugin for another service, then awsRegion for example doesn't necessarily makes any sense.

Reply:
The only reason the `awsRegion` is there is to build the urls which isn't really necessary. The important thing is that we know that the files have been uploaded. I will move this logic to the application using this plugin instead to make it more generic.